README.md

```diff
@@ -167,6 +167,7 @@ Supported variables are as follows:
 - **beats_version** (*Defaults to `7.6.0`*): Beats version.
 - **version_lock** (*Defaults to `false`*): Locks the installed version if set to true, thus preventing other processes from updating. This will not impact the roles ability to update the beat on subsequent runs (it unlocks and re-locks if required).
 - **use_repository** (*Defaults to `true`*): Use elastic repo for yum or apt if true. If false, a custom custom_package_url must be provided.
+- **install_repository** (*Defaults to `{use_repository}`*): Install elastic repo for yum or apt if true. If false, the present repositories will be used. Useful if you already have beats packages in your repo.
```

Comment:
ansible-elasticsearch is using [`es_add_repository`](https://github.com/elastic/ansible-elasticsearch/blob/master/defaults/main.yml#L8) variable. Could you rename `install_repository` to `beats_add_repository` for consistency?

(Note to myself that we still need to prefix ansible-beats variables by `beats_` to ensure no risk of conflicts with variables from other roles)